com.github.TuxPusher.yaml

```diff
@@ -0,0 +1,22 @@
+app-id: com.github.TuxPusher
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: tuxpusher
+rename-icon: tuxpusher
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: tuxpusher
+    buildsystem: simple
+    build-commands:
+      - cd TuxPusher_SDL;make;make PREFIX=${FLATPAK_DEST} install
+      - install -Dm 0644 TuxPusher_SDL/flatpak/tuxpusher.desktop ${FLATPAK_DEST}/share/applications/tuxpusher.desktop
+      - install -Dm 0644 TuxPusher_SDL/flatpak/tuxpusher.appdata.xml ${FLATPAK_DEST}/share/metainfo/tuxpusher.appdata.xml
+      - install -Dm 0644 TuxPusher_SDL/flatpak/tuxpusher.png ${FLATPAK_DEST}/share/icons/hicolor/192x192/apps/tuxpusher.png
```

Comment:
it doesn't matter here, but seeing these being in a `flatpak` directory upstream feels wrong. NONE of this is flatpak specific. the icon space and .destkop are 20+ years old now.